src/libraries/System.Private.CoreLib/gen/System.Private.CoreLib.Generators.csproj

```diff
@@ -5,15 +5,18 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="BitOperations.cs" />
     <Compile Include="ProductVersionInfoGenerator.cs" />
     <Compile Include="EventSourceGenerator.cs" />
     <Compile Include="EventSourceGenerator.Emitter.cs" />
     <Compile Include="EventSourceGenerator.Parser.cs" />
     <Compile Include="IntrinsicsInSystemPrivateCoreLibAnalyzer.cs" />
+    <Compile Include="$(CommonPath)System\Sha1ForNonSecretPurposes.cs" Link="Common\System\Sha1ForNonSecretPurposes.cs" />
```

Comment:
Is this going to require addition to the exception paperwork for uses of Sha1ForNonSecretPurposes?

cc @GrabYourPitchforks

Reply:
Initially I had done the suppression and just let it use `Sha1` but this felt more correct.